client/my-sites/stats/activity-log-confirm-dialog/index.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import Button from 'components/button';
+import Gridicon from 'gridicons';
+
+class ActivityLogConfirmDialog extends Component {
+	static propTypes = {
+		siteName: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		siteName: 'YourAwesomeSite', // FIXME: real site name
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			showDialog: true,
+		};
+
+		this.onCloseDialog = this.onCloseDialog.bind( this );
+	}
```

Comment:
`constructor` is mostly being avoided in Calypso.

Initial state can just be defined like this:
```js
state = { showDialog: true };
```